.github/workflows/compile.yml

```diff
@@ -2,6 +2,16 @@ name: CI
 
 on: [push, pull_request]
```

Comment:
Do we need to restrict this to `main` branch and only on `push`?

Reply:
The deploy job will fail on PRs because it won't get the token.  It's a good idea to run it only on the main branch so that we don't get every PR failing.

Reply:
... which Wes has done already.